### PR TITLE
Simulate doubleclick, since it doesnt works with click simultaneously…

### PR DESCRIPTION
**Pull request by vladest**
… in js

Test plan:
- go to entity list
- click on an entity. make sure its selected
- double click with quite long interval between clicks (> 500ms but less < 1000ms). Make sure entiy focused

Unfortunately, this is the best I can do with JS/HTML. clicks and double clicks doesnt works at same time and following click with interval less than 500ms (approximately) just swallowed

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 77ba8c743f985ebefc3d521e049439cdc422f87a available at: https://gist.github.com/cf7de2715f1ba729c6302857fdb81867<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10216-77ba8c7.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10216-77ba8c7.dmg

File: scripts/system/html/js/entityList.js [hunk 1]
```diff
@@ -41,7 +44,7 @@ function loaded() {
       elRadius = document.getElementById("radius");
       elExport = document.getElementById("export");
       elPal = document.getElementById("pal");
-      elEntityTable = document.getElementById("entity-table");
+      //elEntityTable = document.getElementById("entity-table");
```
Comment (sethalves) [inline scripts/system/html/js/entityList.js]: go ahead and delete the line.

File: scripts/system/html/js/entityList.js [hunk 2]
```diff
@@ -84,8 +87,28 @@ function loaded() {
           setSortColumn('hasScript');
       };
 
+      var clicks = 0, delay = 1000;
+
+      function onMouUp(clickEvent) {
```
Comment (sethalves) [inline scripts/system/html/js/entityList.js]: could you call this onMouseUp so that I find it with grep?

**Comment by hifi-gustavo** [comment]
Build console output for commit: c053d814affcfba7636cfd5ec8ef9fc35fc6d677 available at: https://gist.github.com/cf7de2715f1ba729c6302857fdb81867<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10216-c053d81.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10216-c053d81.dmg

**Comment by Nex-Pro** [comment]
Testing this now.

**Comment by Nex-Pro** [comment]
Dropping this. Unsure what I am looking for.

**Comment by cozza13** [comment]
I could not get this to work by clicking

**Pull request closed without merging**